Chart: update Multiple Axes demo description (#2271)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/MultipleAxes/description.md

> With the Chart component, you can display charts with multiple axes for better data visualization. For example, in a single-axis chart, you have to choose which values to indicate - absolute or percentage. With a multi-axis chart, you can indicate both absolute and percentage values on two separate axes, as you can see in this demo.

Reviewer: Could you please clarify why did you use the word "indicate" in this context? Don't you think the words like "designate" or "mark" would look better here?

Moreover, I can say that this isn't the best possible sentence...

> For example, in a single-axis chart, you have to choose which values to indicate - absolute or percentage.

It sounds too colloquial for our documentation.

Also, you start two sentences in this short paragraph with "With". It doesn't look good as well.

Can you rewrite this paragraph?

To configure a multi-axis chart, follow the steps below:

> 1. Create and name the value axes.         

Reviewer: We do need this extra emphasis here. Just create and name value axes.

```suggestion
1. Create and name value axes.
```

> Declare multiple objects in the [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/) array. Each object configures a single value axis. Then, give each value axis a unique [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name).

Reviewer: A == one, that is, single. When you use both "a single", you are putting a very strong emphasis on singularity. Do we really need it? I have my doubts.

```suggestion
Declare multiple objects in the [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/) array. Each object configures a value axis. Every value axis should have a unique [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name).
```

2. Bind series to value axes.        

> Assign the axis name to the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[axis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#axis) property. If you do not specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[axis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#axis) property, such a series is bound to the axis declared first in the [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/) array.

Reviewer: To bind a series to **any** axis, we should assign the name of an axis. We need the name of any (target) axis.

```suggestion
Assign the name of an axis to the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[axis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#axis) property. If leave the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[axis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#axis) property unspecified, the series is bound to the axis declared first in the [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/) array.
```

> In this demo, the first axis is declared without the [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name) and is bound to the first series. The second axis [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name) is `total` and it is bound to the spline series. The [position](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#position) of the second axis is `right`.

Reviewer: > The second axis [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name) is `total` and it is bound to the spline series.

There is a logical flaw. This sentence in its current state says that the mentioned name is bound to the spline series, not the axis. I do not think we can say that the axis and its name can be used interchangeably in terms of semantics.

```suggestion
In this demo, the first axis is declared without a [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name) and is bound to the first series. The [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name) of the second axis is `total` and it is bound to the spline series. The [position](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#position) of the second axis is `right`.
```
